packages/mock-account-provider/app/lib/accounts.server.ts

```diff
@@ -214,13 +214,4 @@ export class AccountProvider implements AccountsServer {
   }
 }
 
-declare global {
-  let __mockAccounts: AccountsServer | undefined
-}
-
-if (!global.__mockAccounts) {
-  global.__mockAccounts = new AccountProvider()
-}
-const mockAccounts = global.__mockAccounts
-
-export { mockAccounts }
```

Comment:
This was giving me issues:
> Property '__mockAccounts' does not exist on type 'typeof globalThis'

I converted to just exporting out the `mockAccounts` instance. Should give the same behaviour: importing modules will get the same instance on module resolution